apps/okreads-e2e/src/integration/search-books.spec.ts

```diff
@@ -1,17 +1,34 @@
+import { $, $$, browser, ExpectedConditions } from 'protractor';
+import { expect } from 'chai';
+
 describe('When: Use the search feature', () => {
-  beforeEach(() => {
-    cy.startAt('/');
-  });
 
-  it('Then: I should be able to search books by title', () => {
-    cy.get('input[type="search"]').type('javascript');
+  it('Then: I should be able to search books by title', async () => {
+    await browser.get('/');
+    await browser.wait(
+
+    ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okReads')
+    );
 
-    cy.get('form').submit();
+    const form = await $('form');
+    const input = await $('input[type="search"]');
+    await input.sendKeys('javascript');
+    await form.submit();
 
-    cy.get('[data-testing="book-item"]').should('have.length.greaterThan', 1);
+    const items = await $$('[data-testing="book-item"]');
+    expect(items.length).to.be.greaterThan(1, 'At least one book');
   });
 
-  xit('Then: I should see search results as I am typing', () => {
-    // TODO: Implement this test!
+  it('Then: I should see search results as I am typing', async () => {
+    await browser.get('/');
+    await browser.wait(
+      ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okReads')
+    );
+
+    const input = await $('input[type="search"]');
+    await input.sendKeys('j');
```

Comment:
Since the current implementation is already in Cypress , why don't you just leave it the way it is and continue on the top of the same implementation?